modules/index-service/src/main/java/org/opencastproject/index/service/resources/list/provider/ContributorsListProvider.java

```diff
@@ -65,13 +70,30 @@
 
   private static final Logger logger = LoggerFactory.getLogger(ContributorsListProvider.class);
 
+  private Set<String> excludeUserProvider = new HashSet<>();
   private UserDirectoryService userDirectoryService;
   private AbstractSearchIndex searchIndex;
 
   protected void activate(BundleContext bundleContext) {
     logger.info("Contributors list provider activated!");
   }
 
+  public void updated(Dictionary<String, ?> properties) throws ConfigurationException {
+    if (properties == null) {
+      logger.debug("No configuration available, using defaults.");
```

Comment:
This will not load the defaults if you delete the configuraion file.

Reply:
suggestion committed